### PR TITLE
Upgrade docker containers to python 3.6

### PR DESCRIPTION
**Pull request by madchap**
With additional care this time around.

### CODE REVIEW

**Comment by madchap** [comment]
We updated the PR template to now accept 3.6 contributions. Should this then be part of 1.6.1?

**Comment by valentijnscholten** [comment]
> We updated the PR template to now accept 3.6 contributions. Should this then be part of 1.6.1?

don't think so as anything that gets contributed will go into dev/1.7.0 anyway which will be 3.6 then?

File: Dockerfile.nginx [hunk 1]
```diff
@@ -29,7 +31,7 @@ ENV \
   node="nodejs"
 RUN \
   apt-get -y update && \
-  apt-get -y install apt-transport-https ca-certificates && \
+  apt-get -y install apt-transport-https ca-certificates curl wget && \
```
Comment (alles-klar) [inline Dockerfile.nginx:34]: do we really need to install curl and wget? Doesn't seem related to the switch from python 3.5 to 3.6?

Comment (valentijnscholten) [reply Dockerfile.nginx:34]: probablty needed because they are no longer included by default?

Comment (madchap) [reply Dockerfile.nginx:34]: Indeed.

**Comment by ptrovatelli** [comment]
these need an update  , in dockerfile.django:
```
# Using 3.5.7 to avoid compatibility issues that may be introduced by python 3.6 and 3.7.
# Please upgrade before end-of-life in september 2020!
```
